### DISCUSSION TITLE
The features of `proof-general` doesn't seem to be "browsable" in epkg.

### DISCUSSION THREAD

**Discussion by erikmd** [emacscollective/epkg#35] [Support] [answered]
dear @tarsius, dear all,

after the merge of [ProofGeneral/PG#627](https://github.com/ProofGeneral/PG/pull/627#issuecomment-979290805),
I ran `epkg-update` to see if the license change was updated in epkg accordingly.
And, yes 👍

But meanwhile, I've noticed that several features that are part of `proof-general` are not browsable/clickable,

cf. the following screenshot obtained with GNU Emacs 26.1 on GNU/Linux Debian:
![2021-12-06_23-40-32_Screenshot_epkg_proof-general](https://user-images.githubusercontent.com/10367254/144934907-efac1bf5-fd28-4ec3-8927-72e90ef0a926.png)

Would you have an idea/explanation of this (minor) issue?

**Comment by tarsius** [comment, marked as answer]
You can only view libraries that have already been installed. Did you use `borg` to install this package and did you try restarting Emacs?

**Reply by erikmd** [reply]
No, I didn't use `borg`, I use a regular MELPA + `use-package` install (basically, with this [.emacs](https://github.com/erikmd/tapfa-init.el/blob/master/.emacs))

So I was especially using `M-x epkg-list-packages` to benefit from its detailed metadata (not for installing the package)

Thus at first, I was puzzled to see that some features are "detected", and other not…

**Reply by erikmd** [reply]
But I guess that when you said

> You can only view libraries that have already been installed.

You also meant that requiring the features (or opening a buffer attached to the corresponding features) should mark them as "browsable".

And, indeed, it seems there is no "issue" actually 😅

**Reply by erikmd** [reply]
After opening a `coq.v` file and reloading the buffer, I get this:

![2021-12-07_00-06-29_Screenshot_epkg_proof-general_OK](https://user-images.githubusercontent.com/10367254/144937162-e50ccac3-1dd8-4423-a362-64933ef75506.png)

So thanks for your feedback! I guess we can mark the question as closed :)
